src/main/kotlin/no/nav/modiapersonoversikt/skrivestotte/service/LeaderElectorService.kt

```diff
@@ -35,7 +36,7 @@ class LeaderElectorService(val configuration: Configuration) {
     internal fun getLeader(): LeaderElectorResponse {
         return try {
             runBlocking {
-                val response = client.get<String>(configuration.electorPath)
+                val response = client.get(configuration.electorPath).body<String>()
```

Comment:
[NITPICK] Som oppfølging kan vi se på å bytte om til `jackson` for serialisering her, så slipper vi å ha avhengigheter til to serialiserings-bibliotek (e.g fjerne `io.ktor:ktor-serialization-gson`).